extensions/binding/org.eclipse.smarthome.binding.lifx/src/main/java/org/eclipse/smarthome/binding/lifx/fields/Field.java

```diff
@@ -0,0 +1,91 @@
+/**
+ * Copyright  (c) 2014 timothyb89 (https://github.com/timothyb89/lifx-java) under the MIT license
```

Comment:
As Tim agreed to re-license it under EPL, you should change the license headers accordingly (all classes)